pom.xml

```diff
@@ -33,6 +33,7 @@
 		<module>spring-cloud-stream-test-support</module>
 		<module>spring-cloud-stream-test-support-internal</module>
 		<module>spring-cloud-stream-configuration-metadata</module>
+		<module>spring-cloud-stream-implicit-expression-processor</module>
```

Comment:
Does this really require a new module? Generally speaking, while it is helpful at writing modules, it is a configuration enhancement, not specific to Spring Cloud Stream proper.